reflex/utils/exec.py

```diff
@@ -36,9 +37,13 @@ def run_process_and_launch_url(run_command: list[str]):
     )
 
     for line in processes.stream_logs("Starting frontend", process):
-        if "ready started server on" in line:
-            url = line.split("url: ")[-1].strip()
+        if match := re.search("ready started server on ([0-9.:]+)", line):
+            url = match.group(1)
+            if get_config().frontend_path != "":
+                url += get_config().frontend_path
```

Comment:
I'd prefer to see `urllib.parse.urljoin` here, to ensure that leading and trailing slashes are squashed.

```suggestion
                url = urljoin(url, get_config().frontend_path)
```

Reply:
Done.